### PR TITLE
Fix Option Text

### PR DESCRIPTION
**Pull request by SuperManitu**
See #471

### CODE REVIEW

File: dom/README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ the app on the DOM.
 
 - `container: String|HTMLElement` the DOM selector for the element (or the element itself) to contain the rendering of the VTrees.
 - `options: DOMDriverOptions` an object with two optional fields: `transposition: boolean` enables/disables transposition of inner streams in
-the virtual DOM tree, `modules: array` contains additional Snabbdom modules.
+the virtual DOM tree, `modules: array` overrides the default Snabbdom modules.
```
Comment (mightyiam) [inline dom/README.md]: How about a link where I can see what the default Snabbdom modules are?
And it should be also clear, whether these defaults are in Cycle.js or upstream Snabbdom defaults.

Comment (TylorS) [inline dom/README.md]: Good idea @mightyiam. The defaults are to Cycle.js. Snabbdom has no concept of any defaults at present. The defaults for Cycle.js are the `class`, `attrs`, `props`, and `style` modules as an FYI.

**Comment by staltz** [comment]
Hi :)
Thanks for submitting this PR, we appreciate any kind of desire to help. Before merging this PR I need you to notice that this README.md is always generated from the source. So could you please make a PR where we update only the JSDoc comment in the source code? That will be reflected in the generated README.md

**Comment by SuperManitu** [comment]
will do tomorrow

**Pull request closed without merging**